convert2rhel/backup/__init__.py

```diff
@@ -70,6 +70,12 @@ def push(self, restorable):
         if not isinstance(restorable, RestorableChange):
             raise TypeError("`%s` is not a RestorableChange object" % restorable)
 
+        # check if the file is already added
+        for r in self._restorables:
+            if r == restorable:
+                loggerinst.debug("{} is already backed up".format(restorable.backup_path))
```

Comment:
```suggestion
                loggerinst.debug("{} is already backed up".format(r.backup_path))
```
or `loggerinst.debug("{} is already backed up".format(restorable.filepath))`